src/components/WebGLView/WebGLView.tsx

```diff
@@ -1142,6 +1142,7 @@ export const WebGLView = connector(
             ANormalized.get(this.props.colorScales.scales, this.props.pointColorScale as string),
             this.props.channelColor.key,
             this.props.dataset,
+            this.createAdditionalColumns(),
```

Comment:
fixes the bug that when restoring a session the coloring of group labels is not restored